test/integration/content-explorer/ContentExplorer.e2e.test.js

```diff
@@ -23,7 +23,7 @@ describe('ContentExplorer', () => {
 
         it('Navigation within a collection keeps sidebar closed', () => {
             // Manually closing the sidebar should remove its content
-            cy.getByTestId('sidebaractivity').click();
+            cy.getByTestId('sidebarskills').click();
```

Comment:
What's the reason for changing this value?

Reply:
Test was failing for me, and when I debugged it, I found that the 'Sample Audio.mp3' file is defaulting to the skills tab, not the activity feed, so in order to toggle it closed, I had to click the skills tab button